src/main/webapp/data-visualization.js

```diff
@@ -294,13 +301,28 @@ function setStyle(isCountyQuery) {
     const buttonsDiv = document.getElementById('buttons');
     buttonsDiv.style.display = 'none';
   }
-
   const chartsDiv = document.getElementById('am-charts');
   chartsDiv.style.display = 'block';
   const mapsDiv = document.getElementById('map');
   mapsDiv.style.display = 'none';
 }
 
+// Changes the color of the current visualizations on the page. 
+function changeColor(colorParam) {
+  if (typeof map !== 'undefined') {
```

Comment:
In what situation would map be undefined? If it is, should we be throwing an error or something?

Reply:
I haven't implemented the geoJson for state queries, so for state queries maps is undefined. Added comment to clarify this.